examples/pytorch/graphsaint/README.md

```diff
@@ -6,7 +6,7 @@ Paper link: https://arxiv.org/abs/1907.04931
 
 Author's code: https://github.com/GraphSAINT/GraphSAINT
 
-Contributor: Liu Tang ([@lt610](https://github.com/lt610))
+Contributor: Jiahang Li ([@ljh1064126026](https://github.com/ljh1064126026))  Tang Liu ([@lt610](https://github.com/lt610))
 
 ## Dependencies
 
```

Comment:
"you need to rename graphsaintdata to data", why don't we directly modify the code to handle this?